File: Practice/21/Python/bmi.py
def bmi(weight: float, height: float) -> float:
    return weight / (height * height / 10000)
def print_bmi(bmi: float) -> float:
    if bmi < 18.5:
        print('Underweight')
    elif 18.5 <= bmi and bmi < 25:
        print('Normal')
    elif 25 <= bmi and bmi < 30:
        print('Overweight')
    else:
        print("Obesity")
print("Введите вес (в килограммах) и рост (в сантиметрах) человека через пробел")  
w, h = map(float,input().split())
print_bmi(bmi(w, h))

File: Practice/21/C++/bmi/bmi.cpp
#include <iostream>
using namespace std;

double BMI(double weight, double height)
{
	return weight / (height * height / 10000);
}

void printBMI(double BMI)
{
	if (BMI < 18.5)
		cout << "Underweight";
	else if ((18.5 <= BMI) and (BMI < 25))
		cout << "Normal";
	else if ((25 <= BMI) and (BMI < 30))
		cout << "Overweight";
	else
		cout << "Obesity";
}

int main()
{
	setlocale(LC_ALL, "Russian");

	double w, h;
	cout << "Введите вес (в килограммах) и рост (в сантиметрах) человека через пробел\n";
	cin >> w >> h;
	printBMI(BMI(w, h));
}